bin/permadelete_crash_data.py

```diff
@@ -108,18 +110,18 @@ def s3_delete(crashid):
     bucket = s3_context.config.bucket_name
     s3_client = s3_context.client
 
-    # Delete the raw crash
-    key = "v2/raw_crash/%(entropy)s/%(date)s/%(crashid)s" % {
-        "entropy": crashid[0:3],
-        "date": "20" + crashid[-6:],
-        "crashid": crashid,
-    }
-    obj = s3_fetch_object(s3_client, bucket, key)
-    if obj:
-        s3_delete_object(s3_client, bucket, key)
+    keys = build_keys("raw_crash", crashid)
+    for key in keys:
+        try:
+            obj = s3_fetch_object(s3_client, bucket, key)
+            if obj:
+                s3_delete_object(s3_client, bucket, key)
+        except Exception as exc:
+            print(f"exception fetching/deleting raw crash: {key}")
+            print_exception(exc)
 
     # Fetch dump_names which tells us which dumps exist
-    key = "v1/dump_names/%(crashid)s" % {"crashid": crashid}
+    key = build_keys("dump_names", crashid)[0]
```

Comment:
We should probably do the same thing here where we try every key even though we've never moved `dump_names` so it's only ever had one key.